python/src/trezorlib/cli/cardano.py

```diff
@@ -59,14 +60,46 @@ def sign_tx(client, file, protocol_magic):
 
 
 @cli.command()
-@click.option("-n", "--address", required=True, help=PATH_HELP)
+@click.option("-n", "--path-str", required=True, help=PATH_HELP)
```

Comment:
please keep the name `address`

Reply:
Done [here](https://github.com/trezor/trezor-firmware/pull/1112/commits/aa4c7374f5a2985426319d47fe36b58392e7e6f9).